composer.json

```
@@ -3,8 +3,8 @@
     "description": "Serve files to the browser that are not available directly",
     "type": "library",
     "require": {
-        "php": "^5.3.9 | ^7.0",
-        "symfony/http-foundation": "^2.8|^3.0"
+        "php": "^5.3.9 | ^7.0 | ^8.0",
```

Could do without unsupported PHP-versions...
```suggestion
        "php": "^7.4 | ^8.0",
```